config-controller/README.md

```diff
@@ -0,0 +1,89 @@
+# ACS Config Controller
+
+Manage ACS configuration using Kube custom resources
+
+## Getting Started
+
+### Prerequisites
+- go version v1.20.0+
+- docker version 17.03+.
+- kubectl version v1.11.3+.
+- Access to a Kubernetes v1.11.3+ cluster.
```

Comment:
so old :scream:

Reply:
It seems kubebuilder has strong backwards compatibility support :D